src/ui/buttons/buttonDesigns/fadeOutLogoShowLabelText.jsx

```diff
@@ -0,0 +1,121 @@
+/* @flow */
+/** @jsx node */
+import { LOGO_CLASS } from '@paypal/sdk-logos/src';
+import { node, Fragment, type ChildType } from 'jsx-pragmatic/src';
+
+import { CLASS } from '../../../constants';
+import { BUTTON_SIZE_STYLE } from '../config';
+
+import type {  LabelOptions, DivideLogoAnimationProps } from './types';
+
+export const FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG = {
+    tiny:       { min: BUTTON_SIZE_STYLE.tiny.minWidth },
```

Comment:
instead of having nested objects for tiny and medium can you use min and max instead like https://github.com/paypal/paypal-checkout-components/blob/master/src/ui/buttons/buttonDesigns/divideLogoAnimation.jsx#L19-L20

Reply:
I am okay with that